retrofit2-clients/src/main/java/com/palantir/remoting1/retrofit2/MultiServerRetryInterceptor.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.remoting1.retrofit2;
```

Comment:
Question: Is it the right package/subproject? This class and other interceptors from this package do not depend on retrofit2 and can be used with just OkHttpClient